indico/modules/events/templates/display/conference/base.html

```diff
@@ -42,8 +46,10 @@ <h1>
                     <div class="confSubTitle f-self-stretch" style="{{ conf_layout_params.text_color_css }}">
                         {{ template_hook('conference-header', event=event) }}
                         <div class="datePlace">
-                            <div class="date">{{ _format_event_date(event) }}</div>
-                            <div class="place">{{ event.venue_name }}</div>
+                            <div class="date">
+                                 {{ _format_event_date(event) }}
```

Comment:
why not keep this in one line? also, too much indentation ;)

Reply:
Fixed, was a leftover from having things inlined here instead of using the macro above.